lib/loader.js

```diff
@@ -43,6 +43,12 @@ function sassLoader(content) {
         addNormalizedDependency
     ));
 
+    // Skip empty files, otherwise it will stop webpack, see issue #21
+    if (options.data.trim() === "") {
+        callback(null, "");
```

Comment:
Noooo, I don't like that style (and my global eslint complains about inconsistent return values) 😁